source/_integrations/azure_data_explorer.markdown

```diff
@@ -0,0 +1,188 @@
+---
+title: Azure Data Explorer
+description: Setup for Azure Data Explorer integration
+ha_category:
+  - History
+ha_release: 2024.6
+ha_config_flow: true
+ha_iot_class: Cloud Push
+ha_codeowners:
+  - '@kaareseras'
+ha_domain: azure_data_explorer
+---
+[Azure Data Explorer](https://azure.microsoft.com/en-us/services/data-explorer/)  is a high performace timeseries database, query engine and dashboarding tool. The Home Assistant `Azure Data Explorer` integration allows you to hook into the Home Assistant event bus and forward events to Azure Data Explorer for analytics and dashboarding. From here data can be viewed in buildin dashboards, PowerBi and Grafana amoung others.
+
+## Create a free Azure account
+
+* Create a  [free Azure account](https://azure.microsoft.com/). you will be asked for creditcard info, but all rescources created here are free.
+
+## Create a Service Principal (App registration)
+
+For Home Assistant to authenticate with Azure Data Explorer, it need a **Service Principal**
+1. Create a [Service Principal](https://docs.microsoft.com/en-us/azure/data-explorer/provision-azure-ad-app) follow guide step 1-7
```

Comment:
```suggestion
1. To create a Service Principal, follow the guide on [Creating Microsoft Entra application registration](https://docs.microsoft.com/en-us/azure/data-explorer/provision-azure-ad-app) steps 1-7.
```
This link leads to the *Create Microsoft Entra application registration* procedure. Is this correct?
I added the name of the procedure to the step, to make it clear that this is the procedure to follow.